vite-base.config.ts

```diff
@@ -61,8 +61,12 @@ export default defineConfig({
   server: {
     port: 8080
   },
+  /**
+   * Some old libs like `promise-queue` and `readable-stream` still uses Webpack.
+   */
```

Comment:
```suggestion
   // Some old libs like `promise-queue` and `readable-stream` still uses Webpack.
```

This is not a JSDoc, neither a long comment, so you can use inline comment.

Reply:
https://github.com/Adamant-im/adamant-im/pull/800/commits/4dc3ab3cb3bef9b390ed4c18bc7ea617ad926cc5